doc/source/multiple-sources.rst

```diff
@@ -0,0 +1,36 @@
+=======================
+ Multiple Data Sources
+=======================
+
+Data Files
+==========
+
+Part details, indexed by a part number:
+
+.. include:: part-details.yaml
+   :literal:
+
+Inventory counts:
+
+.. include:: inventory.csv
+   :literal:
+
+Template File
+=============
+
+.. include:: _templates/inventory.tmpl
+   :literal:
+
+Loading the Template
+====================
+
+.. code-block:: rst
+
+   .. datatemplate:csv:: inventory.csv
```

Comment:
Is it possible to *only* load data with the load() statement?

If so, add an inline example?

``` 
.. datatemplate::

   {% set data = load('inventory.csv') %}
   ...
```

Reply:
Looks good BTW!

Reply:
It would be possible to do that using the `nodata` loader. https://github.com/sphinx-contrib/datatemplates/blob/master/doc/source/nodata.rst

Reply:
I woke up thinking about that.